temba_client/exceptions.py

```diff
@@ -56,7 +56,8 @@ def __str__(self):
 
 
 class TembaSerializationException(TembaException):
-    pass
+    def __init__(self, message):
+        self.message = message
```

Comment:
I wanted to call `super()` but I wasn't sure what Python version y'all supported, and then I noticed that other `TembaException` subclasses that overrode `__init__()` never called `super()`, so I avoided it too... 🤷‍♂

Reply:
We dropped support for Python 2 a while back but never made the code more Python 3-ish